api/package.json

```diff
@@ -122,6 +122,7 @@
 		"exif-reader": "2.0.1",
 		"express": "4.20.0",
 		"flat": "6.0.1",
+		"fluent-ffmpeg": "2.1.3",
```

Comment:
`fluent-ffmpeg` requires `ffmpeg >= 0.9` to already be installed on the host system. To ship this we'd need to at least include `ffmpeg` in the docker image and probably also need some additional docs/instructions for development setups.